recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
   host:
     - python
     - pip
-    - patch
+    - patch  # [not win]
```

Comment:
why so?  it is 'patch' for unix, 'm2-patch' for windows

Reply:
test:
  imports:
    - sacrebleu
  requires:
    - pip
  
lacks 'python <3.10'